client/src/components/EventView/Header/index.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FiPrinter, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import InputEditable from "../../Common/InputEditable";
+import DateEditable from "../../Common/DateEditable";
+import Stack from "../../Common/Stack";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing8,
+  spacing16,
+  sideMargins
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  background-color: ${(props) => props.bgcolor || theme1};
+  padding: 0 ${sideMargins};
+`;
+
+const TitleContainer = styled.div`
+  width: 60%;
+`;
+
+const TextContainer = styled.div`
+  height: 100%;
+  width: calc(100% - ${spacing8});
+  margin-right: ${spacing16};
+  font-size: ${fontSize64};
+`;
+
+const DateContainer = styled.div`
+  font-size: ${fontSize24};
+`;
+
+const DateInput = styled.div`
+  opacity: 0.5;
+  margin: 0;
+`;
+
+const ButtonBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * The header which appears at the top of the event-edit page.
+ */
+const Header = ({
+  event: {
+    title,
+    date: { startDate }
+  },
+  // dateDisplay, TODO, add this in after the first demo
+  backgroundColor,
+  onPressDownload,
+  onPressPrint,
+  ...props
+}) => {
+  const realDate = new Date(startDate);
+  return (
+    <Container {...{ bgcolor: backgroundColor, ...props }}>
+      <Stack>
+        <TitleContainer>
+            <TextContainer>{title}</TextContainer>
+        </TitleContainer>
+        <DateContainer>
+          <DateInput
```

Comment:
We don't want the date to be editable, so you would want to make your own component to just display the date